prime-router/src/main/kotlin/azure/CheckFunction.kt

```diff
@@ -150,14 +155,14 @@ class CheckFunction : Logging {
         @BindingName("orgName") orgName: String,
         @BindingName("receiverName") receiverName: String
     ): HttpResponseMessage {
-        val claims = AuthenticatedClaims.authenticate(request)
-
-        if (claims == null || !claims.authorized(setOf("*.*.primeadmin", "$orgName.*.admin"))) {
-            logger.warn("User '${claims?.userName}' FAILED authorized for endpoint ${request.uri}")
-            return HttpUtilities.unauthorizedResponse(request, authenticationFailure)
-        }
-
-        logger.info("User ${claims.userName} authorized for endpoint ${request.uri}")
+        // val claims = AuthenticatedClaims.authenticate(request)
```

Comment:
This code authenticates the website's API request. Did you mean to add it back in before the PR?

Reply:
Yes, code was commented out for local testing - restored.